internal/venice-common/src/main/java/com/linkedin/venice/pubsub/manager/TopicMetadataFetcher.java

```diff
@@ -69,7 +69,7 @@ class TopicMetadataFetcher implements Closeable {
    * implementations are not guaranteed to be thread-safe.
    */
   private final BlockingQueue<PubSubConsumerAdapter> pubSubConsumerPool;
-  private final List<Closeable> closeables;
```

Comment:
There is a reason why we use private `List<Closeable> closeables`; instead of `List<PubSubConsumerAdapter> pubSubConsumerAdapters`;.
	1.	We want to enforce that PubSubConsumerAdapter instances are always retrieved from the `pubSubConsumerPool`, which is a `BlockingQueue`. Introducing a separate list could encourage direct access and usage, leading to potential concurrent use of consumer clients. (Refer to the comment on `pubSubConsumerPool` for more context.)
	2.	Although `closeables` currently only contain consumer adapters, they are designed to potentially hold other resources in the future that also need to be properly closed
when stopping the topic manager.